### PR TITLE
Make population a required variable and remove the default query

### PR DESCRIPTION
**Pull request by rebkwok**

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -36,3 +39,24 @@ def spark_database(containers):
...
+
+
+@pytest.fixture
+def OldCohortWithPopulation():
+    class OldCohort:
+        def __init_subclass__(cls):
+            if not hasattr(cls, "population"):
+                cls.population = table("practice_registrations").exists()
+
+    yield OldCohort
```
Comment (evansd) [inline tests/conftest.py:51]: How come this needs to be a fixture? Couldn't you get the same effect by just importing this class into each of the test modules and then you wouldn't need to include it in every single test function signature.

Comment (rebkwok) [reply tests/conftest.py:51]: It could; in fact I did it like that before and can't remember why I changed it.

Comment (rebkwok) [reply tests/conftest.py:51]: I think it may have just been that I was doing the new version as a fixture; I'll change it back

**Review by evansd — APPROVED** [review]
Great, looks good - thanks!

**Pull request merged**